test/helpers/fixture.ts

```diff
@@ -68,6 +68,7 @@ export const defaultTestConfig: InitTestInput = {
 
   stakingArguments: {
     minValidatorStakingAmount: BigNumber.from(100),
+    maxCommissionRate: 100_00,
```

Comment:
```suggestion
    maxCommissionRate: 20_00, // 20%
```

Reply:
This is fixture config for test suite. Ignore.